Cython/Utility/ModuleSetupCode.c

```diff
@@ -829,6 +829,13 @@ static CYTHON_INLINE PyObject * __Pyx_PyDict_GetItemStrWithError(PyObject *dict,
   #define PySet_CheckExact(obj)        __Pyx_IS_TYPE(obj, &PySet_Type)
 #endif
 
+
+#if PY_VERSION_HEX >= 0x030900A4
+  #define _Pyx_SET_SIZE(obj, size) Py_SET_SIZE(obj, size)
+#else
+  #define _Pyx_SET_SIZE(obj, size) do { Py_SIZE(obj) = (size); } while (0)
```

Comment:
The prefix is `__Pyx_`, not `_Pyx_`.
Also, the `do…while` isn't needed since we control the usages on our side.

Reply:
Oops, I fixed both issues.